src/components/Toolbar.tsx

```diff
@@ -22,6 +22,8 @@ import ToolbarButton from './ToolbarButton'
 import TriangleLeft from './TriangleLeft'
 import TriangleRight from './TriangleRight'
 
+const ANIMATION_MS = 80
```

Comment:
Hoist this into `constants.ts` so that `ToolbarButton` can use it. That will help avoid one getting accidentally changed without the other.

You can give it a more globally unique name, like `TOOLBAR_PRESS_ANIMATION` and add a JSDOC comment that explains its usage.

Reply:
Good suggestion! Implemented here: 60c641e5cde817ac6bbc56e1255e266e0b35a4eb